ptf_tests/common/utils/test_utils.py

```diff
@@ -1017,7 +993,249 @@ def host_check_netperf(remote=False, hostname="",username="",password=""):
         print (f"FAILED: faild to pkill -9 netserver")
         connection.tear_down()
         return False
-    print (f"PASS: pkill -9 netserver and ready to restart netserver on {hostname}")
+    print (f"PASS: \"pkill -9 netserver\" and ready to restart netserver on {hostname}")
+    
+    connection.tear_down()
+    return True
+
+def get_vm_interface_counter(conn, interface):
+    """
+    A funtion to build a counter directory for a VM interface statics
+    """
+    counter ={}
+    cmd= f"ip -s -s link show dev {interface}"
```

Comment:
-s twice. is this correct?

Reply:
It works. More information provided 

[root@wp26 install]# ip -s -s link show dev eno1
2: eno1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP mode DEFAULT group default qlen 1000
    link/ether a4:bf:01:3e:1e:44 brd ff:ff:ff:ff:ff:ff
    RX: bytes  packets  errors  dropped overrun mcast
    12045185209 8544775  0       0       0       52905
    RX errors: length   crc     frame   fifo    missed
               0        0       0       0       0
    TX: bytes  packets  errors  dropped carrier collsns
    143657346  1380950  0       0       0       0
    TX errors: aborted  fifo   window heartbeat transns
               0        0       0       0       2
    altname enp61s0f0

[root@wp26 install]#
[root@wp26 install]# ip -s link show dev eno1
2: eno1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP mode DEFAULT group default qlen 1000
    link/ether a4:bf:01:3e:1e:44 brd ff:ff:ff:ff:ff:ff
    RX: bytes  packets  errors  dropped overrun mcast
    12045202149 8545027  0       0       0       52905
    TX: bytes  packets  errors  dropped carrier collsns
    143741828  1381367  0       0       0       0
    altname enp61s0f0